chromeapp/app/scripts/dnssd/dns-packet.js

```diff
@@ -0,0 +1,401 @@
+/*jshint esnext:true, bitwise:false */
+
+/**
+ * Represents a DNS packet.
+ *
+ * The structure of the packet is based on the information in 'TCP/IP
+ * Illustrated, Volume 1: The Protocols' by Stevens.
+ */
+'use strict';
+
+var resRec = require('./resource-record');
+var dnsCodes = require('./dns-codes');
+var byteArray = require('./byte-array');
+var qSection = require('./question-section');
+
+var MAX_ID = 65535;
```

Comment:
Consider adding comments for these constants and what they mean/where they come from.